src/identity/contactMapping.ts

```diff
@@ -243,6 +275,43 @@ function* fetchWalletAddresses(e164Number: string) {
   }
 }
 
+function* fetchAddressVerification(address: string) {
+  try {
+    const walletAddress = yield* select(walletAddressSelector)
+    const signedMessage = yield* call(retrieveSignedMessage)
+
+    const addressVerificationQueryParams = new URLSearchParams({
+      address,
+      clientPlatform: Platform.OS,
+      clientVersion: DeviceInfo.getVersion(),
+    }).toString()
+
+    const response: Response = yield* call(
```

Comment:
not sure if we do this for other identity service requests, but should these have a timeout?

Reply:
Not sure; we don't appear to do that for other calls in this file.

Reply:
I think it's good practice to have timeouts, otherwise the request can hang for a long time. Maybe we can create a follow up ticket / PR to add timeouts?